### PR TITLE
Migrate exercise w02 programs to Scala 3, fix #541

### LINKED ISSUE #541: Migrate exercise w02 programs to Scala 3

### PR DESCRIPTION
**Pull request by kuleshmaria**
New:
indentation subtask in exercise 4
javap from terminal in exercise 18

### CODE REVIEW

File: quiz/QuizData.scala [hunk 1]
```diff
@@ -70,8 +70,8 @@ object QuizData {  // to generate tables for a concept connection quizes in late
       "kompilerad     " -> "maskinkod sparad och kan köras igen utan kompilering",
       "skript         " -> "maskinkod sparas ej utan skapas vid varje körning",
       "objekt         " -> "samlar variabler och funktioner",
-      "main           " -> "där exekveringen av kompilerat program startar",
-      "programargument" -> "överförs via parametern args i main",
+      "@main          " -> "där exekveringen av kompilerat program startar",
+      "programargument" -> "kan överföras via parametern args i en primitiv main",
```
Comment (bjornregnell) [inline quiz/QuizData.scala:74]: You can also give params to an @main functions. So remove "primitiv" and perhaps better with "till" instead of "i"

**Review by bjornregnell — CHANGES_REQUESTED** [review]

**Comment by kuleshmaria** [comment]
Fixed!
I thought based on the discussion about the Scala 3 installation the other day.
I tested the exercises in the chapter with a Scala 3 installation from binaries, but the commands for compiling and REPL are a little different with Coursier?

**Comment by bjornregnell** [comment]
Yes unfortunately they are called `scala3-repl` and `scala3-compiler` and there is no app runner (!) -- I have hopes they will fix it so let's wait a bit before we change everything. I think `cursier` needs some significant resources or we need a better scala runner that can hide coursier in the background or something. Esp. support on Windows is bad which is bad. Let's wait and see until mid august. Let me know when you think this is ready for merge again.

**Comment by kuleshmaria** [comment]
Hope so, otherwise I think it is ready for merge!

**Pull request merged**